tests/unit/libbpf_test.cpp

```diff
@@ -2372,6 +2368,9 @@ TEST_CASE("libbpf attach type names", "[libbpf]")
 
     enum bpf_attach_type attach_type;
     for (int i = 1; i < __MAX_BPF_ATTACH_TYPE; i++) {
+        // Skip XDP type as it is supported only with the xdp-for-windows repo.
+        if (i == BPF_XDP)
+            continue;
```

Comment:
Maybe ok for step 1 but hard coding individual values here is not a good solution long term as we want to someday have lots of types supported by other repos.